src/evaluation/actions.py

```diff
@@ -103,6 +104,12 @@ def wait_for_element(self, input: Input) -> Result:
 
         return Result(success=True, outcome=input_element, action=f"self.wait_for_element({input})")
 
+    def clear_text(self, action: ActionChains):
```

Comment:
Should this be moved to `ActionUtils`? I am hoping that we keep `MyActions` as the collections of actions that a language model can take.

Reply:
Possibly, I never thought of that. Though it may be beneficial to pass it gestures as well like "clear_text" since it's not the most obvious that in key presses you have to hold down control, press a, and then let go of control. Like it may be a lot easier to infer to type certain text since we are just sending it characters to type, but not "complex" interactions like this. But if we want the model to be able to do this, then yeah that makes sense!

Reply:
Moved into ActionUtils. I didn't know that was an option.

I have a suggestion regarding future complex actions like "clear_text" since they require the ctrl key to be held down and used in conjunction of another operation (ctrl down, send a, ctrl up) that may be difficult for a model to learn possibly? Might not be a bad idea to define a complex_actions class in `MyActions` and pass in a string like clear_text which will run the ActionUtils function

@danyaljj